packages/discovery-provider/plugins/pedalboard/apps/solana-relay/src/index.ts

```diff
@@ -20,6 +20,13 @@ import { location } from './routes/instruction/location'
 import { listen } from './routes/listen/listen'
 import { relay } from './routes/relay/relay'
 
+process.on('unhandledRejection', (reason, promise) => {
+  logger.error(
+    { promise, reason, stack: reason instanceof Error ? reason.stack : null },
```

Comment:
Fwiw I think you can pass the reason as the first arg and pino will auto format it as an error if it is one: https://github.com/pinojs/pino/blob/main/docs/api.md#mergingobject-object